CHANGELOG.md

```diff
@@ -12,6 +12,14 @@ In the next version (v0.28.0) it will be fully removed from the database.
 
 ### Added
 
+#### Moderated content can now be removed from search index
+PR [\#8811](https://github.com/decidim/decidim/pull/8811) is addressing an issue when the moderated resources are not removed from search index.
+We have introduced a new task  that would remove the moderated content from being displayed in search. Please run the following command if you want to remove old moderated data from your search results
```

Comment:
```suggestion
This will automatically work for new moderated resources. For already existing ones, we have introduced a new task that will remove the moderated content from being displayed in search:
```